modules/monitoring/files/usr/lib/nagios/plugins/check_aws_rds_memory

```diff
@@ -65,8 +65,8 @@ import boto3
 #################################
 parser = argparse.ArgumentParser()
 parser.add_argument("-r","--region", type=str, choices=['us-east-1','eu-west-1'], help="AWS region to check")
-parser.add_argument("-w","--warning", type=float, help="Value(GB) at which to raise a warning alert (5.0)", default=5.0)
-parser.add_argument("-c","--critical", type=float, help="Value(GB) at which to raise a critical alert (2.0)", default=2.0)
+parser.add_argument("-w","--warning", type=float, help="Value(GB) at which to raise a warning alert (50.0)", default=50.0)
```

Comment:
These values need to be lower (We are trying to avoid a current spurious error in MySQL.. and you had a look at PG from what I gather from the card).. Can you please change this to 2.0GB for warning and 1.0GB for critical?

Reply:
I understand but my only problem with this is that the command line check seems to the through strange errors when I use this value, so, for example, it does not trigger an alert :( 

It only seems to work to a specific decimal point. I have done the change to fix staging alerts.